packages/api/package.json

```diff
@@ -27,6 +27,7 @@
     "jsonwebtoken": "^9.0.0",
     "lodash": "^4.17.21",
     "minimist": "^1.2.7",
+    "mongodb": "^6.3.0",
```

Comment:
I think this is supposed to be dev dep ?

Reply:
It's actually a peer dep of `connect-mongo`, not sure why it was working before: https://github.com/jdesboeufs/connect-mongo/blob/master/package.json#L61

Reply:
from mongoose 🤷‍♂️ . also the peer seems to be `^4.1.0`.  anyway as long as it doesn't introduce deps issue